scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -178,10 +207,6 @@ task LoadData {
 
   command <<<
     set -x
-    set +e
-    # make sure dataset exists
-    bq ls --project_id ~{project_id} ~{dataset_name} > /dev/null
```

Comment:
Does this still fail with a nice error message if there's no dataset? (Nice enough that I'll realize I forgot to make one)

Reply:
I removed this because later on when you go to make the table, it fails will a clear error message if the dataset doesn't exist